tidb-cloud/set-up-private-endpoint-connections.md

```diff
@@ -79,6 +79,10 @@ Your service endpoint region is selected by default. Have a quick check and clic
 
 ### Step 3. Create an AWS interface endpoint
 
+> **Note:**
+>
+> The endpoint service of each dedicated cluster will be automatically created for newly created after March 28, 2023.
+
 TiDB Cloud begins creating an endpoint service, which takes 3 to 4 minutes.
 
 When the endpoint service is created, take a note of your endpoint service name from the command in the lower area of the console.
```

Comment:
```suggestion
If you see the `Endpoint Service Ready` message, take note of your endpoint service name from the command in the lower area of the console for later use. Otherwise, wait for 3 to 4 minutes to let TiDB Cloud create an endpoint service for your cluster.
```